WordPress/src/main/java/org/wordpress/android/ui/people/PeopleManagementActivity.java

```diff
@@ -97,4 +105,26 @@ public void onJSONException(JSONException exception) {
             }
         });
     }
+
+    @Override
+    public void onPersonSelected(Person person) {
+        FragmentManager fragmentManager = getFragmentManager();
+        PersonDetailFragment personDetailFragment = (PersonDetailFragment) fragmentManager.findFragmentByTag(KEY_PERSON_DETAIL_FRAGMENT);
+
+        long personID = person.getPersonID();
+        int localTableBlogID = person.getLocalTableBlogId();
+        if (personDetailFragment == null) {
+            personDetailFragment = PersonDetailFragment.newInstance(personID, localTableBlogID);
+        } else {
+            personDetailFragment.setPersonDetails(personID, localTableBlogID);
+        }
+        if (!personDetailFragment.isAdded()) {
+            FragmentTransaction fragmentTransaction = getFragmentManager().beginTransaction();
+            fragmentTransaction.setCustomAnimations(R.animator.fragment_slide_in_from_right, R.animator.fragment_slide_out_to_left,
```

Comment:
I think the issue is caused by a bug in the SDK per this thread: https://code.google.com/p/android/issues/detail?id=25994 Even though the issue is created for support fragments, further down in the comments it's mentioned that it's actually an issue for the native Fragments as well.

I've found a workaround in that thread which is implemented in [#9298551](https://github.com/wordpress-mobile/WordPress-Android/pull/3970/commits/9298551ec46462a1dba86c11f4cfa7adcb71b179). Basically instead of using the `FragmentTransaction` for setting the animation, we're using the `onCreateAnimator` inside the `PersonDetailFragment` which does basically the same thing and it works even after the config changes.

Another alternative was to add `android:configChanges="orientation|screenSize"` to the `AndroidManifest` for `PeopleManagementActivity` but I don't think we'd want to do that just for this. Also, I don't think it would work if the user taps Home and gets back to the app.

All in all, I think the final solution is pretty good and I don't think there is any issues left with the animation.